CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## 3.0.0 - UNRELEASED
+
+- Documents and prioritizes the existing `apos` command alias as the CLI
+command.
+- More colors! Command logs are no longer all an ❗️alarming❗️ shade of red. Red now means errors.
+
+### Breaks
+
+- The Apostrophe CLI no longer supports running core and Apostrophe module tasks.
+  - The CLI tool was providing an alias for these `apos apostrophe-users:add admin admin`. These can all still be run more directly on the command line with `node app`, e.g., `node app apostrophe-users:add admin admin`.
+- Creating a piece type with `create-piece` no longer attempts to convert a singular piece type name to plural. The piece type name should be provided as plural for A2 or singular for A3, as per the custom.
```

Comment:
Erm. So the custom in A2 is that the `name` option gets set singular but the module name gets set plural. Which was terrible and I'm glad it's over in A3 and maybe we should just change it as a convention for new work in A2 (it's not like A2 insists we continue to follow this custom in order to work), but it is the custom.

Reply:
Piece type module names are plural in A2, so the idea here is that you provide the piece name as plural. We're removing the pluralization that was in the CLI tool, which is the reason for this. Between pluralization working differently in other languages and it working inconsistently in English the win isn't really so huge. A3 adds more complication, so I'm getting us out of it in the CLI completely.

Reply:
I'm pointing out that if we really want to stick to A2's pattern we then have to perform singularization before setting the "name" option. We can say "ugh, let's just not," but then people have to know that when they write joins; specifying withType in the singular won't work, which may be an A2 expectation at this point.

Reply:
:ballmer: